.github/container/Dockerfile.base

```diff
@@ -17,30 +17,37 @@ ARG DEST_MANIFEST_DIR
 ## Install Python and essential tools
 ###############################################################################
 
-RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive \
-    TZ=America/Los_Angeles \
-    apt-get install -y \
-        build-essential \
-        checkinstall \
-        clang \
-        cmake \
-        curl \
-        git \
-        lld \
-        vim \
-        bat \
-        curl \
-        git \
-        gnupg \
-        rsync \
-        python-is-python3 \
-        python3-pip \
-        liblzma-dev \
-        wget \
-    && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN <<"EOF" bash -ex
+export DEBIAN_FRONTEND=noninteractive
+export TZ=America/Los_Angeles
+apt-get update
+apt-get install -y curl
+
+# Repo for newer nsight versions
+curl -o /usr/share/keyrings/nvidia.pub https://developer.download.nvidia.com/devtools/repos/ubuntu2204/$(dpkg --print-architecture)/nvidia.pub
+echo "deb [signed-by=/usr/share/keyrings/nvidia.pub] https://developer.download.nvidia.com/devtools/repos/ubuntu2204/$(dpkg --print-architecture)/ /" > /etc/apt/sources.list.d/devtools-ubuntu2204-$(dpkg --print-architecture).list
```

Comment:
Can we parameterize the `2204` part so that we don't have to manually update it when bumping the base OS version?
That'll also be more friendly if we want to matricize the build over other Ubuntu versions.

Reply:
Done.